ambry-clustermap/src/main/java/com.github.ambry.clustermap/Partition.java

```diff
@@ -186,6 +186,12 @@ public JSONObject toJSONObject()
     return jsonObject;
   }
 
+  /**
+   * Generates a {@link String} representation of this {@code Partition} that uniquely identifies the
+   * {@code Partition}. The string representation is in the format of {@code Partition[i]},
```

Comment:
I think you have used the @code tags wrong here. These should be @link annotations at best but even that need not be done because it is self referencing.